docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+services:
+  db:
+    image: postgres
```

Comment:
@rakshans1 Can we use `postgres:11.3-alpine` for smaller download size?

Reply:
@BilalBudhani Yes we can. Let me test it and will update